resume-buddy/src/main/java/com/google/sps/data/Reviewee.java

```diff
@@ -1,15 +1,32 @@
 package com.google.sps.data;
 
-/*TODO: add more information from form input into Reviewee*/
 public final class Reviewee {
 
   private final String firstName;
   private final String lastName;
   private final String email;
+  private final String school;
```

Comment:
I recommend storing the school as a lowercase string with leading and trailing whitespaces trimmed, which will make matching easier. If someone enters "Cornell University" and " cornell university " they should be the same string. 

Also interesting to think about if/how you'll be able to match someone who input "Cornell" with someone who put "Cornell University" and all the string variations of a school.

Reply:
I was also thinking about that and instead of guessing how to match two university name strings, we could make a drop down selection and populate with university names. We could find a list of universities somewhere or scrape it from website like this https://www.4icu.org/us/a-z/

Reply:
I'm gonna check that out Ani! In a new PR im gonna try to import a drop down of all university names from somewhere. I'll add a TODO for now